.strata.yml

```diff
@@ -0,0 +1,12 @@
+pipeline_template: jar/Jenkinsfile-1
+email_reply_to: mc-hypergage@salesforce.com
+time_out_mins: 180                                   # (Optional) default shown
+number_of_artifacts_to_keep: 3                       # (Optional) default shown
+compliance_required: false                           # (Optional) default shown
+docker_test_images:
+   - dva/sfdc_centos7_java8_build
+unit_tests_command: echo "no tests included"
+production_branch:                                   # (Optional) default shown
+     - v2.2.0-evg                                    # if using regex, append "r:" in front of pattern
+email_only_last_committer_on_dev_branch: true        # (Optional) default shown
+publish_jar_image: dva/hypergage-jdk11-gradle:7b5e9d6d3b2d792981601501186872c806e1086e
```

Comment:
Does this one have to built w/ JDK11 too? Even though javac source/target are set to 8, there is some risk of JDK11 API usage sneaking in.

Reply:
It does not, was the path of least resistance.  I tested it and it seemed ok but if you insist I will build on JDK8.

Reply:
I guess as long as the evergage-product ITs pass, it's probably ok. We have pretty good tests for the gpg stuff.